pkg/kubelet/lifecycle/handlers.go

```diff
@@ -55,9 +59,16 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
 		var buffer bytes.Buffer
 		output := ioutils.WriteCloserWrapper(&buffer)
 		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false)
+		if err != nil {
+			glog.V(1).Infof("Exec lifecycle hook for Container %q in Pod %q failed - %q", container.Name, format.Pod(pod), buffer.String())
```

Comment:
Sorry I don't think I was clear. We should print out the content of `handler.Exec.Command` so that it's obvious what hook failed.